app/views/support/support.html.erb

```diff
@@ -1,20 +1,22 @@
-<% set_page_title('Support') %>
+<% set_page_title(title_with_error_prefix('Support', error: @support_form.errors.any?)) %>
 <% set_page_description('How to get support for GOV.UK Forms - a new platform for people who work in government to create online forms for GOV.UK') %>
 
-<h1 class="govuk-heading-xl">Support</h1>
-<p>
-  GOV.UK Forms is a new platform for people who work in government to create online forms for GOV.UK.
-</p>
-
-<p class="govuk-inset-text">
-  GOV.UK Forms does not provide support for individual members of the public about government forms. If you are a member of the public experiencing difficulties with a government form, please <a href="https://www.gov.uk/contact">contact that service directly</a>.
-</p>
-
-<h2 class="govuk-heading-m">How to contact us</h2>
-
-<p>
-  If you have a question about GOV.UK Forms you can contact us at <a href="mailto:govuk-forms@digital.cabinet-office.gov.uk">govuk-forms@digital.cabinet-office.gov.uk</a>.
-</p>
+<%= form_with model: @support_form, url: new_support_message_path do |f| %>
+  <% f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-xl">Support</h1>
+  <p>
+    GOV.UK Forms is a new platform for people who work in government to create online forms for GOV.UK.
```

Comment:
Can we put this into the translation file for consistency?

Reply:
Done in [8de4cbd](https://github.com/alphagov/forms-product-page/pull/75/commits/8de4cbdbbf530a11298d3c1fb04c012029036bc4)